src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -119,16 +58,12 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 	private _isFilteringDirectories: boolean = false;
 	private _mostRecentCompletion?: ISimpleCompletion;
 
-	private _codeCompletionsRequested: boolean = false;
-	private _gitCompletionsRequested: boolean = false;
-
 	// TODO: Remove these in favor of prompt input state
 	private _leadingLineContent?: string;
 	private _cursorIndexDelta: number = 0;
 
-	private _lastUserDataTimestamp: number = 0;
-	private _lastAcceptedCompletionTimestamp: number = 0;
 	private _lastUserData?: string;
+	static lastAcceptedCompletionTimestamp: number = 0;
 
 	isPasting: boolean = false;
 
```

Comment:
The `requestCompletionsSequence`, etc. statics are pwsh specific and should live in the pwsh file

Reply:
oh, they do. forgot to delete these.